### ISSUE TITLE
Weak attack regardless of strength

### ISSUE THREAD

**Issue by ezekeal** [HabitRPG/habitica#4195]
I can't seem to do much damage to Bosses, It's stuck at around 1 to 4 no matter how much my strength has gone up. Same with my girlfriend's character. I'm not sure what the reasonable range is per daily/to-do, but it seems like it's not changing at all.

Other members who started months later seem to be doing more 15 to 35

User ID: 73ce8dbe-cf39-4143-8077-15f76a5d6ca0
Her user ID: ac66c7b4-c3ae-430e-a1eb-9b35554e77c1
Group ID: d7a4e2a7-e92a-4235-a621-7334601e7210

**Comment by Alys** [comment]
Relevant stats from both accounts:

```
{
    "_id": "73ce8dbe-cf39-4143-8077-15f76a5d6ca0",
    "stats": {
        "class": "warrior",
        "con": 10,
        "exp": 375,
        "int": 3,
        "lvl": 25,
        "per": 4,
        "points": 0,
        "str": 8,
        "buffs": {
            "str": 0,
            "int": 0,
            "per": 0,
            "con": 0,
            "stealth": 0,
            "streaks": false
        }
    },
    "party": {
        "order": "random",
        "quest": {
            "key": "atom2",
            "completed": null,
            "progress": {
                "up": 4.549524316989,
                "down": 0,
                "collect": {}
            }
        }
    },
    "items": {
        "gear": {
            "equipped": {
                "shield": "shield_warrior_3",
                "head": "head_special_2",
                "armor": "armor_healer_3",
                "weapon": "weapon_warrior_4"
            },
        }
    }
}


{
    "_id": "ac66c7b4-c3ae-430e-a1eb-9b35554e77c1",
    "stats": {
        "class": "rogue",
        "con": 2,
        "int": 7,
        "lvl": 27,
        "per": 13,
        "points": 0,
        "str": 5,
        "buffs": {
            "str": 14,
            "int": 20,
            "per": 17,
            "con": 24,
            "stealth": 0,
            "streaks": false
        }
    },
    "party": {
        "order": "pets",
        "quest": {
            "key": "atom2",
            "completed": null,
            "progress": {
                "up": 3.8183180101937233,
                "down": 0,
                "collect": {}
            }
        }
    },
    "items": {
        "gear": {
            "equipped": {
                "shield": "shield_rogue_6",
                "head": "head_rogue_5",
                "armor": "armor_rogue_5",
                "weapon": "weapon_rogue_6"
            },
        }
    }
}
```

I think it's likely that this isn't really a bug. Different players do different amount of damage depending on how they use HabitRPG. You both have a relatively small proportion of your attribute points allocated to strength; the other members might have more allocated to strength, or might be using equipment that gives higher strength boosts. The others could also be doing more tasks each day, or have more of their tasks set to a harder difficulty, or have more checklist items on their To-Dos, or be ticking off more red To-Dos. They could also be using [Skills](http://habitrpg.wikia.com/wiki/Skills) that do boss damage.

Try finding more To-Dos that you can action each day, or break big To-Dos into multiple steps with checklists - that will do more boss damage. However the best way to use HabitRPG is to focus on whether it's helping you achieve your goals in real life. Have you been accomplishing more of the things in your life that really matter? If not, think about the things that mean the most to you - projects, hobbies, interpersonal roles, etc - and try to find ways of integrating those things into your task lists. Can you make To-Dos each day that will help you move forward on each of your projects or hobbies? Can you create tasks for spending more time with people you cherish? Can you do advanced planning for projects and put a lot of future To-Dos on your list now, so that when they are due they are dark red and do more boss damage (rewarding you for forward planning)?

**Comment by ezekeal** [comment]
Ok, I wasn't sure if those numbers were adding up, we have the highest strength in the group, but may still be relatively low. It could be not making as many sub to-dos and tasks. 

Thanks for looking, I had no idea how to pull the relevant information out.

**Comment by taldin** [comment]
Your base damage only comes from Dailies and To-Dos.   The higher quality a Daily is (towards blue) the less damage you do with it, so clicking on a bright blue Daily may do less than 1 point of damage.  You will do more damage if you crit, but without a crit, the stuff that you do regularly (the blue Dailies) isn't much effort for you, so the game treats it like a minor hit.    High strength doesn't increase the damage that you do; it increases your likelihood of crits happening.   The amount of crit bonus damage is based on your Con, not your Strength (I think...).      Red Dailies are more of an effort, so will do more damage when you do them, -- but to get them towards red means _not_ doing them, and that means taking Daily damage.   Not worth it.   :)   Like Lady Alys says, multi-checklist To-Dos will do more damage as checklists multiply damage as if each checklist item was a single task.     Of course, being a Warrior, try using Brutal Smash on blue Dailies after checking them off for the day; that's your boss damaging ability.

**Comment by deilann** [comment]
Strength does increase boss damage by .4% a point actually

**Comment by taldin** [comment]
Ah, I thought that was changed, but looking at the code, Ryan is right.   The formula is 
next_daily_val_(1 + (user._statsComputed.str / 200)).   next_daily_val is the new value of the Daily, and  ranges from 3.35 (or so) for a dark red task to 0.57 for a bright blue task.   So if you have a bright blue task and 50 points of Strength (buffs included), you'll do 0.57_(1+(50/200)= 1.25), or 0.7125. 

Doesn't look like crit affects boss damage either.

**Comment by ezekeal** [comment]
Thanks. I wasn't sure how to use brutal smash. I had been using them on tasks before checking them off.
